### ISSUE TITLE
Allow space after forward slashes

### ISSUE THREAD

**Issue by teddybradford** [moczolaszlo/postcss-inline-comment#4]
I get an error with the following code:
`// display: flex;`
but this works fine:
`//display: flex;`
It would be nice if the first example worked too. Many text editors insert a space by default when you comment a line.

**Comment by moczolaszlo** [comment]
I'm affraid, but it's impossible now, because postCSS can't parse that yet. :(

Maybe @ai can refute me... :)

**Comment by ai** [comment]
Yes, we need to wait Custom Syntaxes :-)

**Comment by ScottSmith95** [comment]
Custom syntaxes are here, so is this a possibility again?

**Comment by ai** [comment]
@ScottSmith95 it is already done with [postcss-scss](https://github.com/postcss/postcss-scss#inline-comments-for-postcss)

**Comment by ScottSmith95** [comment]
Seems like that is a much larger plugin than I need for this feature. Also, not sure if it's broken with gulp-postcss, but it doesn't seem to run as a plugin.

**Comment by ai** [comment]
@ScottSmith95 `postcss-scss` is not a plugin, it is a parser. You should set it to `parser` option in `process` method. I created a issue in `gulp-postcss` to add syntax options to docs.

**Comment by ScottSmith95** [comment]
So what is the way to declare this (in the meantime)?

```
 var processors = [
    parser(require('postcss-scss')),
    require('postcss-import')({skipDuplicates: false}),
    require('postcss-custom-properties'),
    require('postcss-mixins'),
    require('postcss-nested'),
    require('postcss-custom-media'),
    require('autoprefixer')('last 4 versions', '> 1%', 'ie 9', 'ie 8', 'Firefox ESR'),
    require('css-mqpacker')({sort: true})
  ];
```

(My current process array)

**Comment by ai** [comment]
@ScottSmith95 it is option, not plugin ;)

``` js
var postcss = require('gulp-postcss');
var scss    = require('postcss-scss');

var plugins = […];

return gulp.src('src/*.css')
    .pipe( postcss(plugins, { parser: scss }) )
```

**Comment by ScottSmith95** [comment]
Thanks! I'd still submit that a custom parser seems a bit scary for what seems like a small feature, although I know it's hard to parse without a custom parser.
